docs/cookbook/entities/first-resource.rst

```diff
@@ -46,6 +46,44 @@ As an example we will take an **Article entity**.
         }
     }
 
+
+If you don't add a form type, it uses a `default form type`_. But it is a good practice to have one.
+
+.. code-block:: php
+
+    // src/Form/Type/ArticleType.php
+
+    namespace App\Form\Type;
+
+    use Symfony\Component\Form\AbstractType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
```

Comment:
It's not used here. My bad I think.

Reply:
lol. I added one for testing. I will remove it.